tests/kola/kdump/crash/test.sh

```diff
@@ -16,7 +16,7 @@ set -xeuo pipefail
 
 case "${AUTOPKGTEST_REBOOT_MARK:-}" in
   "")
-      if [ $(systemctl show -p Result kdump.service) != "Result=success" ]; then
+      if ! is_service_active kdump; then
```

Comment:
optional:

```suggestion
      if ! is_service_active kdump.service; then
```

I think this might more clear we're talking about systemd service by  adding the `.service` here (and modifying the function).